spec/jobs/rapid_ready_for_decision/form526_hypertension_job_spec.rb

```diff
@@ -145,10 +163,12 @@
       end
 
       it 'finishes successfully' do
-        Sidekiq::Testing.inline! do
-          expect do
-            subject.perform_async(submission_without_account.id)
-          end.not_to raise_error
+        VCR.use_cassette('evss/claims/claims_without_open_compensation_claims') do
```

Comment:
There's a lot of repetition of this line. Wondering if there's a way to set the cassette for a `context` block.

Reply:
Here are some solutions to remedy the repetition: https://gist.github.com/myronmarston/2377461